src/pages/Login.js

```diff
@@ -85,9 +31,12 @@ export default function Login(props) {
           throw e
         }
       }
+    // TODO: add error handling on the request and display errors
+    // TODO: set Redux store values for the user
     if (response.status === 200) {
-      console.log(response)
       setLoginStatus(true)
+      setSuccess(true)
+      localStorage.setItem("id", response.data.user.id) // TODO: change to stored in Redux
       navigate('/landing')
     }
     else {
```

Comment:
This actually refers to line 53 below, but it's not apart of the code you changed so I cannot comment on it:

On my VS "event" is displayed as deprecated. Try passing event in as a parameter to function handleSubmit()?